packages/react-charting/src/components/DonutChart/Arc/Arc.tsx

```diff
@@ -80,6 +111,59 @@ export class Arc extends React.Component<IArcProps, IArcState> {
     wrapTextInsideDonut(classNames.insideDonutString, this.props.innerRadius! * 2 - TEXT_PADDING);
   }
 
+  private _getTruncatedText(text: string, maxWidth: number): string {
```

Comment:
How does this work along with wrapTextInsideDonut.

Reply:
wrapTextInsideDonut ensures that the words are placed in a new line if the current line length exceeds the maximum allowable width inside the donut. However, if a single word itself (even when placed in a new line) exceeds the maxWidth, it does not truncate the word. Thus, _getTruncatedText() determines if truncation is required for a single word or a group of words when inner text exceeds the maximum allowable width.